modules/react/tabs/lib/TabsList.tsx

```diff
@@ -43,33 +44,30 @@ export const useTabsList = composeHooks(
   useListResetCursorOnBlur
 );
 
-const StyledStack = styled(Stack)<StyledType>({
-  '::after': {
-    content: '""',
-    position: 'sticky',
-    height: 52,
-    minWidth: 30,
-    background: `linear-gradient(to right,rgba(255,255,255,0),white);`,
-    zIndex: 1,
-    right: 0,
-    top: 0,
-    pointerEvents: 'none',
-  },
-});
+const StyledStack = styled(Stack)<StyledType & {maskImage?: string}>(({maskImage}) => ({
+  maskImage: maskImage,
+}));
 
 export const TabsList = createSubcomponent('div')({
   displayName: 'Tabs.List',
   modelHook: useTabsModel,
   elemPropsHook: useTabsList,
 })<TabListProps>(({children, overflowButton, ...elemProps}, Element, model) => {
   const modality = useModalityType();
+  const isRTL = useIsRTL();
   return (
     <StyledStack
       as={Element}
       position="relative"
       borderBottom={`1px solid ${commonColors.divider}`}
-      paddingX={modality === 'touch' ? 'zero' : 'm'}
+      paddingLeft={modality === 'touch' ? 'zero' : 'm'}
+      paddingRight="m"
```

Comment:
We added this `paddingRight` to give the fade out gradient space to occupy so it doesn't show up on top of the last tab (see screenshot [here](https://github.com/Workday/canvas-kit/pull/1705#discussion_r938342058)). However, it doesn't seem to show up for me when I view the Basic Tabs story on my phone:

![IMG_2223](https://user-images.githubusercontent.com/146020/191564554-6296cfda-a7de-4d2e-9012-c9a3ee99fd34.jpg)

Strangely, it does show up on desktop Chrome in device mode:

<img width="482" alt="Screen Shot 2022-09-21 at 9 52 22 AM" src="https://user-images.githubusercontent.com/146020/191564974-28eb32e3-97a1-4431-ae16-03513106c883.png">

Reply:
This appears to be fixed in iOS 16. Sharing a video from @jaclynjessup which compares iOS 15 vs iOS 16 using the Xcode Simulator:

https://user-images.githubusercontent.com/146020/191618273-43034ce5-992e-4595-8c51-7b49558e2332.mov

Note the `paddingRight1` shows up in iOS 16 but not in iOS 15.